README.md

```diff
@@ -190,6 +190,21 @@ end
 
 Scripttags are created in the same way as the Webhooks, with a background job which will create the required scripttags.
 
+If you want to use the application.js file as generated by the Rails asset pipeline, you can do so by moving the `ShopifyApp.configure`
```

Comment:
Ah, I reread. Hrm... Are you suggesting sending all the bundles generated by the asset pipeline? I think one would typically send one bundle, no?